localgovdrupal/localgov_microsites_group
# Feature/235 webforms

This should be ready to test!

Steps to test.

Fresh install, enable localgov_microsites_group_webform

`lando drush si localgov_microsites -y; lando drush en localgov_microsites_group_webform -y ; lando drush cr; lando drush uli; 
`

Then: 

1. Create a controller user, log in as controller
2. Create microsite 1 and microsite 2
3. Create 2 trusted editor users and assign to each microsite as a microsite admin
4. Log into microsite 1 as admin 1
5. On site 1 as admin 1, create a microsite webform node, with overrides for submission email and confirmation message.
6. On site 2 as admin 2, create a microsite webform node, with overrides for submission email and confirmation message.
7. As anonymous, create submissions on each webform
8. As admin 1 on site 1 confirm I can see the submissions for my site and not see the submissions for other sites
9. As admin 2 on site 2, confirm I can see the submissions for my site and not see the submissions for other sites.
